### DISCUSSION TITLE
Support scram-sha-256 in PostgreSQL ?

### DISCUSSION THREAD

**Discussion by fredericalix** [ovh/cds#5611] [Q&A] [answered]
Hi dream deam o/
Do you plan to support scram-sha-256 in PostgreSQL in the next CDS release ?

++, fax

**Comment by yesnault** [comment]
could be, It's now supported by lib/pq since https://github.com/lib/pq/releases/tag/v1.1.0

**Comment by shueja** [comment]
I’ve asked in #5619 for workarounds in database setup. Is Postgres 13 not compatible at the moment?

**Comment by yesnault** [comment]
next on https://github.com/ovh/cds/issues/5644

**Comment by yesnault** [comment, marked as answer]
> Do you plan to support scram-sha-256 in PostgreSQL in the next CDS release ?
Available in release 0.48.0
